### PR TITLE
Add enable WATVAP 

### PR DESCRIPTION
**Pull request by goncalvesmachadoc**

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: python/cxx/eclipse_config.cpp [hunk 1]
```diff
@@ -36,5 +36,6 @@ void python::common::export_EclipseConfig(py::module& module)
         .def("useCPR",               &SimulationConfig::useCPR )
         .def("hasDISGAS",            &SimulationConfig::hasDISGAS )
         .def("hasVAPOIL",            &SimulationConfig::hasVAPOIL );
+        .def("hasVAPWAT",            &SimulationConfig::hasVAPWAT );
```
Comment (bska) [inline python/cxx/eclipse_config.cpp:39]: You need to remove the terminating semicolon (`;`) on the `"hasVAPOIL"` line.

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks for the updates.  This looks good to me now and I'll merge into master.

**Pull request merged**